tbapy/main.py

```diff
@@ -387,3 +409,78 @@ def district_teams(self, district, simple=False, keys=False):
             return self._get('district/%s/teams/keys' % district)
         else:
             return [Team(raw) for raw in self._get('district/%s/teams' % district)]
+
+    def update_trusted(self, auth_id, auth_secret, auth_event_key):
+        """
+        Set Trusted API ID and Secret and the event key they are assigned to.
+
+        :param auth_id: Your event authorization ID, obtainable at https://www.thebluealliance.com/request/apiwrite
+        :param auth_secret: Your event authorization secret, obtainable at https://www.thebluealliance.com/request/apiwrite
+        """
+        self.auth_id = auth_id
+        self.auth_secret = auth_secret
+        self.auth_event_key = auth_event_key
+
+    def update_event_info(self, data):
+        """
+        Update an event's info on The Blue Alliance.
+
+        :param data: Dictionary of data to update the event with.
+        """
+        return self._post('event/%s/info/update', json.dumps(data))
```

Comment:
Do you think it would be better to have the user specify the event key inside each function or as an instance variable? (Also, it might be advantageous to change the name of the variable from auth_event_key to event_key anyway. Why "auth"?) Just wondering what you think @NikhilNarayana and @ErikBoesen

Reply:
I used auth_event_key because each auth_id and auth_secret are linked to a single event so I believe it makes the most sense to have the event_key set for any instance of the class. The variable name isn't that big of a concern to me, when I put "auth" in the name I was thinking of authorized event key.

Reply:
I decided to just change it to event_key and update the doc strings. 4b070151e41a52aed9eac3e6bacdc800b4895521